test/unit/latch.c

```diff
@@ -1,6 +1,8 @@
 #include "memory.h"
```

Comment:
How it looks in `git log`:

![image](https://user-images.githubusercontent.com/92749/193938220-b8bb4b65-2f4d-458d-85d2-bf9565617892.png)

BTW, shouldn't the commits be attached to some issue?

Reply:
> BTW, shouldn't the commits be attached to some issue?

That was weak attempt to avoid thousands of references in the #7618 leading to stale commits, which would be replaced and force-pushed after then...

Now, I've restored links of `Part of #7618`.